### ISSUE TITLE
Missing tests

### ISSUE THREAD

**Issue by buildscientist** [sjaveed/acts_as_nps_rateable#2]
Nice library. It's in dire need of some tests though.

**Comment by sjaveed** [comment]
Agreed.  Tests are in the pipeline and will be needed specially for any PRs coming through.  I'll keep this issue open until I've started specs.
